en/docs/WIP/APIs/api-categories.md

```diff
@@ -0,0 +1,47 @@
+NOTE : THIS FEATURE IS AVAILABLE IN 3.1.0 ONLY
```

Comment:
Hope this is a new feature available from 3.1.0 ? shall we rephrase it
@msm1992 @Mariangela